File: dayEight/java/ArraysOfIntegers/TaskOne.java
public class TaskOne {

    public static boolean isEven(int number) {
       return number % 2 == 0;
    }
    public static boolean isOdd(int number) {
        return number % 2 != 0;
    }
    public static int[] AddOneToEveryOddAndMultiplyEveryEvenIn(int[] numbers) {
        int[] newArray = new int[numbers.length];
        
        for(int index = 0; index < numbers.length; index++){
            if(isEven(numbers[index])) {
                newArray[index] = numbers[index] * 2;
            }
            else if(isOdd(numbers[index])){
                newArray[index] = numbers[index] + 1;
            }
        }
        return newArray;
    }
}

File: dayEight/javaScript/TaskOne.js
    function isEven(number) {
       return number % 2 == 0;
    }
    function isOdd(number) {
        return number % 2 != 0;
    }
    function AddOneToEveryOddAndMultiplyEveryEvenIn(numbers) {
        let newArray = [];
        
        for(let index = 0; index < numbers.length; index++){
            if(isEven(numbers[index])) {
                newArray.push(numbers[index] * 2);
            }
            else if(isOdd(numbers[index])){
                newArray.push(numbers[index] + 1);
            }
        }
        return newArray;
    }

let numbers = [2,8,14,5,1];
console.log(AddOneToEveryOddAndMultiplyEveryEvenIn(numbers))
